climada/hazard/storm_europe.py

```diff
@@ -81,13 +82,32 @@ class StormEurope(Hazard):
     vars_opt = Hazard.vars_opt.union({'ssi_wisc', 'ssi', 'ssi_full_area'})
     """Name of the variables that aren't need to compute the impact."""
 
-    def __init__(self):
-        """Calls the Hazard init dunder. Sets unit to 'm/s'."""
-        Hazard.__init__(self, HAZ_TYPE)
-        self.units = 'm/s'
-        self.ssi = np.array([], float)
-        self.ssi_wisc = np.array([], float)
-        self.ssi_full_area = np.array([], float)
+    def __init__(self,
+                 units: str = 'm/s',
+                 ssi: Optional[np.ndarray] = None,
+                 ssi_wisc: Optional[np.ndarray] = None,
+                 ssi_full_area: Optional[np.ndarray] = None,
+                 **kwargs):
+        """Initialize a StormEurope object
+
+        Parameters
+        ----------
+        units : str
+            The units of the Hazard intensity. Defaults to 'm/s'
+        ssi : numpy.ndarray
+            The Storm Severity Index (SSI). Shape ``(?? x ??)``. Defaults to an empty
+            array.
+        ssi_wisc : numpy.ndarray
+            The Storm Severity Index (SSI) as reconded in the footprint files.
+            Shape ``(?? x ??)``. Defaults to an empty array.
+        ssi_full_area : numpy.ndarray
+            The Storm Severity Index (SSI) ???. Shape ``(?? x ??)``. Defaults to an empty
+            array.
+        """
+        Hazard.__init__(self, haz_type=HAZ_TYPE, units=units, **kwargs)
```

Comment:
Actually, why not `Super.__init__` here? There might be a good reason so I'm curious.

Reply:
No particular reason I guess. `super()` would be cleaner: you could change the super class by changing the one line of code that declares the class. But I don't expect `StormEurope` to inherit from anything else then `Hazard` ever.